### ISSUE TITLE
Travis is failing

### ISSUE THREAD

**Issue by fturib** [coredns/alternate#5]
I think need a sync-up of vendor CoreDNS or Prometheus ... 
(the same issue was fixed in CoreDNS recently)

see : https://travis-ci.org/coredns/alternate

```
github.com/coredns/coredns/coremain
github.com/coredns/coredns/plugin/metrics
# github.com/coredns/coredns/plugin/metrics
../coredns/plugin/metrics/metrics.go:43:49: too many arguments in call to prometheus.NewProcessCollector
	have (int, string)
	want (prometheus.ProcessCollectorOpts)
make: *** [get] Error 2
The command "make" exited with 2.
```

**Comment by fturib** [comment]
hum... I restarted the build in Travis in order to check (last test was 7d old)

**Comment by fturib** [comment]
still in error

```
github.com/prometheus/common/expfmt
github.com/prometheus/client_golang/prometheus
github.com/prometheus/client_golang/prometheus/promhttp
make: *** [get] Error 2
The command "make" exited with 2.
Done. Your build exited with 1.
```

**Comment by fturib** [comment]
fixed with #8
